### ISSUE TITLE
How do Tips work?

### ISSUE THREAD

**Issue by owocki** [gitcoinco/gitcoinco#10]
# Send Tip

When you go to https://gitcoin.co/tip 

![screen shot 2017-12-18 at 10 58 21 am](https://user-images.githubusercontent.com/513929/34120308-65d35774-e3e2-11e7-875a-02d4570654f0.png)

and send a tip,

![screen shot 2017-12-18 at 10 58 40 am](https://user-images.githubusercontent.com/513929/34120313-6b145dd2-e3e2-11e7-89b8-3a85fd250eef.png)

a notification is sent to the users github email addres.

Note:  There is a current issue [where we sometimes cannot pull an email address from github](https://github.com/gitcoinco/web/issues/75) due to permissions issues.  We are working on a long term fix.  In the meatnime, if you're missing your email, please ask on slack.

# Receive Tip

The receive tip page looks like this 

![screen shot 2017-12-18 at 11 00 13 am](https://user-images.githubusercontent.com/513929/34120367-a558ab60-e3e2-11e7-8b41-add7c90145b9.png)

You DO NOT need gas or ether to receive a tip.

This is because Gitcoin creates an ephemeral address that is loaded with _just enough gas_ to process the receipient's transaction.  The hope is that this massively improves the evangelism of the ETH ecosystem... since you do not have to buy gas to participate.

# Any questions?

Post below or ask on [slack](https://gitcoin.co/slack).

**Comment by owocki** [comment]
From a chat convo:

```
Out of interest, do you think that "This is because Gitcoin creates an ephemeral address that is loaded with just enough gas" - there's a potential edge case where the minimum accepted gas price could change and the funds could be locked up? i.e. just enough isn't enough any more... probably very unlikely to happen though - especially given the short duration of a tip.
```
```
yes this is certainly a concern.  esp with the network gasprices fluctutating..  this needs to be handled
```

**Comment by PixelantDesign** [comment]
@mbeacom ok to close?

**Comment by mbeacom** [comment]
@PixelantDesign I think this issue is meant to be a helpful reference versus a bug/feature request.  @owocki ?

**Comment by owocki** [comment]
this is meant to be a helpdesk article. not an active issue we are tracking to close

**Comment by PixelantDesign** [comment]
Gotya. What do you all think about consolidating into how to docs? Could be part of developer and funder guides that we will be taking out of google docs and putting on the site.

**Comment by PixelantDesign** [comment]
Yep we did the same at my previous place. It was just Wordpress though.
